packages/diagrams/frontend/sirius-components-diagrams/src/renderer/edge/CommonEdge.tsx

```diff
@@ -0,0 +1,113 @@
+/*******************************************************************************
```

Comment:
I feel like CommonEdge should still be MultiLabelEdge and that your current MultiLabelEdge  should be called something like SmartEdgeWrapper or EdgeWrapper since it is only used to calculate the props passed to CommonEdge.

Reply:
Yes, you're right.
I've renamed it to `EdgeWrapper`